uniflow/op/model/model_config.py

```diff
@@ -92,3 +92,18 @@ class BedrockModelConfig:
     model_server: str = "BedrockModelServer"
     # TODO: Need to consider the best approach for handling model arguments
     model_kwargs: Dict[str, Any] = field(default_factory=lambda: {})
+
+
+@dataclass
+class SageMakerModelConfig:
+    """Bedrock Model Config Class."""
+
+    endpoint_name: str
+    model_type: str
```

Comment:
Now I am using model_type as a type to prepare different types of input for Falcon or Mistral 7B, since the name of the SageMaker endpoint might not follow a specific pattern.